web/app/scripts/src/providers/project-config.es.js

```diff
@@ -1,28 +1,29 @@
 import { angular } from '../libraries';
 
-const ProjectConfig = function () {
-    let settings = {
-        apiRoot: 'localhost:8000'
-    };
-
-    return {
-        $get: function () {
-            return {
-                getSettings: () => settings
-            }
-        },
-        setApiRoot: (value) => {
-            settings.apiRoot = value;
-        }
-    };
-};
-
-export const module = {
+export const moduleProperties = {
     moduleName: 'incuna.surveys-config',
     componentName: 'ProjectConfig'
 };
 
-angular.module(module.moduleName, [])
-    .provider(module.componentName, [ProjectConfig]);
+const module = angular.module(moduleProperties.moduleName, []);
+
+module.provider(moduleProperties.componentName, [
+    function () {
+        let settings = {
+            apiRoot: 'localhost:8000'
+        };
+
+        return {
+            $get: function () {
+                return {
+                    getSettings: () => settings
```

Comment:
I think it'd be simpler to `return settings` in `$get`. I see how this allows us to add more to it, but I doubt there would be more to add.